vercel/file_path.go

```diff
@@ -6,13 +6,13 @@ import "strings"
 // This is useful as Vercel doesn't allow a root directory that navigates upwards.
 // But if we trim all file paths upwards, and we also trim the root directory, then
 // all is good.
-func trimFilePath(path string) string {
+func trimFilePath(path string) *string {
 	for strings.HasPrefix(path, "../") {
 		path = strings.TrimPrefix(path, "../")
 	}
 	if path == "" {
-		return "."
```

Comment:
I thought (incorrectly) that we allowed `.` as a root_directory. It turns out the correct thing to do is not to specify it as `.`, but to pass `nil`.